include/coap2/coap_io.h

```diff
@@ -51,6 +61,14 @@ typedef struct coap_socket_t {
   coap_fd_t fd;
 #endif /* WITH_LWIP */
   coap_socket_flags_t flags;
+  struct coap_session_t *session; /* Used for the epoll logic if a session.
+                                     Note: It must mot be wrapped with HAVE_SYS_EPOLL_H as
+                                     coap_socket_t is seen in applications embedded in
+                                     coap_session_t etc. */
+  struct coap_endpoint_t *endpoint; /* Used for the epoll logic if an endpoint.
```

Comment:
s/if/of/?

Reply:
Actually `if` is correct. However have changed the comment to `Used by the epoll logic if a session.`